test/prow/fuse_recovery.py

```diff
@@ -0,0 +1,287 @@
+    for name in pods_name:
+        if name.__contains__(prefix):
+            pod_name = name
+            return pod_name
+    return None
+
+
+def checkCsiRecoverEnabled() -> bool:
+    """
+    check if csi-nodeplugin-fluid-xxxx pod.spec.containers has args "FuseRecovery=true"
+    """
+    fluid_namespace = "fluid-system"
+    pod_name = "csi-nodeplugin-fluid"
+    pod_name = getPodNameByPrefix(pod_name, fluid_namespace)
+    if pod_name is None:
+        return False
+    api = client.CoreV1Api()
+    for i in range(10):
+        pod = api.read_namespaced_pod(pod_name, fluid_namespace)
+        if str(pod.spec.containers).__contains__("FuseRecovery=true"):
+            print("CSI recovery enabled")
+            return True
+        time.sleep(5)
+    return False
+
+
+def createDatasetAndRuntime():
+    api = client.CustomObjectsApi()
+    my_dataset = {
+        "apiVersion": "data.fluid.io/v1alpha1",
+        "kind": "Dataset",
+        "metadata": {"name": "hbase"},
+        "spec": {
+            "mounts": [{"mountPoint": "https://mirrors.bit.edu.cn/apache/zookeeper/stable/", "name": "hbase"}]
+        }
+    }
+
+    my_alluxioruntime = {
+        "apiVersion": "data.fluid.io/v1alpha1",
+        "kind": "AlluxioRuntime",
+        "metadata": {"name": "hbase"},
+        "spec": {
+            "replicas": 1,
+            "tieredstore": {
+                "levels": [{
+                    "mediumtype": "MEM",
+                    "path": "/dev/shm",
+                    "quota": "2Gi",
+                    "high": "0.95",
+                    "low": "0.7"
+                }]
+            }
+        }
+    }
+
+    api.create_namespaced_custom_object(
+        group="data.fluid.io",
+        version="v1alpha1",
+        namespace=namespace,
+        plural="datasets",
+        body=my_dataset,
+    )
+    print("Created dataset.")
+    api.create_namespaced_custom_object(
+        group="data.fluid.io",
+        version="v1alpha1",
+        namespace=namespace,
+        plural="alluxioruntimes",
+        body=my_alluxioruntime
+    )
+    print("Created alluxioruntime.")
+
+
+def checkDatasetBound():
+    api = client.CustomObjectsApi()
+    while True:
+        resource = api.get_namespaced_custom_object(
+            group="data.fluid.io",
+            version="v1alpha1",
+            name="hbase",
+            namespace=namespace,
+            plural="datasets"
+        )
+        print(resource)
+        if "status" in resource:
+            if "phase" in resource["status"]:
+                if resource["status"]["phase"] == "Bound":
+                    time.sleep(5)
+                    return True
+        time.sleep(1)
+
+
+def checkFuseRecovered():
+    api = client.CoreV1Api()
+    while True:
+        items = api.list_namespaced_event(namespace=namespace).items
+        for item in items:
+            if item.message.__contains__("Fuse recover"):
+                print("Fuse Recovered.")
```

Comment:
Event is a kind of K8s resource will exist for a long time. So consider the following steps:
1. you create a dataset
2. you delete a fuse pod to trigger fuse recovery
3. Fluid successfully recovered a FUSE mountpoint from dataset. That is, Fluid sends an event containing `Fuse recover`
4. you delete the dataset
5. you recreate a dataset with the same name
6. again, you delete a fuse pod to trigger fuse recovery
7. **At this time, the code above may get phantom events from the last deleted dataset, so the `if` statement returns true and goes on. but the new dataset has not been recovered.**